.github/workflows/kata-deploy-push.yaml

```diff
@@ -47,12 +47,23 @@ jobs:
         uses: actions/download-artifact@v2
         with:
           name: kata-artifacts
-          path: kata-artifacts
+          path: build
       - name: merge-artifacts
         run: |
-          ./tools/packaging/kata-deploy/local-build/kata-deploy-merge-builds.sh kata-artifacts
+          make merge-builds
       - name: store-artifacts
         uses: actions/upload-artifact@v2
         with:
           name: kata-static-tarball
           path: kata-static.tar.xz
+
+  make-kata-tarball:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: make kata-tarball
+        run: |
+          # This will take a while
+          # Test as this is the usual a way will use it for first time.
+          make kata-tarball
+          sudo make install-tarball
```

Comment:
What does the make install end up doing?

Reply:
> What does the make install end up doing? I assume you're just verifying that the make and make install complete successfully.
Yeah , it is just to verify both not fail, I use this locally when I build kata locally with this command so I want to ensure at least is not broken. Verify that it also works is something that would be nice to add in the future.

>  we're not generating artifacts necessarily for consumption. Is that right?
We dont today, but artifacts may be really helpful for those cases in the future, at least what you can do today is go to the action history and get a previous version of kata (for bisect maybe).